### DISCUSSION TITLE
unwrap_parallel.csh

### DISCUSSION THREAD

**Discussion by Congzhangg** [gmtsar/gmtsar#502] [Q&A]
hello everyone
        After I conducted the SBAS command, I found that there were large vacancies in the deformation rate graph. Therefore, I adjusted the unwinding correlation coefficient and ran the unwinding command again. Why did it prompt : 
2 incremental costs clipped to avoid overflow (0.000%)
4 incremental costs clipped to avoid overflow (0.000%)
![20221021205626](https://user-images.githubusercontent.com/112800803/197200997-9e67c375-9e32-4c7b-9b15-83f48dac17c8.png)

**Comment by Xiaohua-Eric-Xu** [comment]
I am not sure. Never saw this before. Did the program finish and produce a proper unwrap.grd?

**Reply by Congzhangg** [reply]
Yes, it managed to get better unwinding data, I guess it doesn't matter

**Reply by Xiaohua-Eric-Xu** [reply]
Yeah, sometimes programs love to complain, but they still finish their jobs.
